diox/pubspec.yaml

```
@@ -0,0 +1,20 @@
+name: sentry_diox
+description: An integration which adds support for performance tracing for the Diox package.
+version: 6.19.0
+homepage: https://docs.sentry.io/platforms/dart/
+repository: https://github.com/getsentry/sentry-dart
+issue_tracker: https://github.com/getsentry/sentry-dart/issues
+
+environment:
+  sdk: '>=2.12.0 <3.0.0'
+
+dependencies:
+  diox: ^5.0.0-dev.3
```

If possible, please use 
```suggestion
  diox: ^5.0.0-0
```
That should make it compatible with all v5 version and stable v5 versions of diox.